app/classifier/annotation-renderer/svg.jsx

```diff
@@ -197,6 +196,7 @@ export default class SVGRenderer extends React.Component {
           </svg>
         </div>
         {(showFeedback) && (<SVGToolTipLayer getScreenCTM={this.getScreenCurrentTransformationMatrix} />)}
+        {this.props.children}
```

Comment:
Why doesn't this render the SVG behind everything else, preventing you from seeing your marks during drawing tasks?

Reply:
I'm not exactly sure. I've been spending some time to understand what the various children are at different levels of the viewer tree and it's hard to keep track. What I have found is that having the `this.props.children` back on line 164 with a z-index will get it to render on top and won't interfere with the drawing SVG layer too. I think it has to do with not totally understanding what's getting priority with the box model and the `position: absolute` being used on both the div with the `svg-subject` class, the audio player, and the svg progress marker. It's confusing to me to have the audio player rendered as a child of the SVGRenderer component too.

Reply:
Thanks for taking a look. The positioning here puzzles me too. This change resets the classifier back to the way it originally worked before I messed about with it in #3995. I can't understand why drawing marks rendered above the subject in the old classifier setup. CSS `position` declarations reset [the stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context), so it may have something to do with that.

Reply:
Oh and the audio player is just plain broken. If you add a subject which is only an mp3 file, the player renders underneath the subject metadata and favourite buttons, and is totally useless.

Reply:
I suppose that can be fixed as needed. Do we have any audio only projects? The projects that want audio at the moment also seem to want the spectrogram image paired with it.